src/Zoomba/Helpers/AppiumCommon.py

```diff
@@ -69,7 +69,9 @@ def drag_and_drop_by_offset(self, locator, x_offset=0, y_offset=0):
     """
     element = self._element_find(locator, True, True)
     zoomba.log('Dragging element "%s" by offset (%s, %s).' % (locator, x_offset, y_offset))
+    x_center = element.location['x'] + element.size['width'] / 2
+    y_center = element.location['y'] + element.size['height'] / 2
```

Comment:
Element.location is taken from the top right corner of the element. To get the center we move in half the height and width. We need this center as that is where the code will click when it performs the initial press.